### PR TITLE
fix: bound realtime websocket state for long sessions

### LINKED ISSUE #3101: 即時轉錄一段時間後會自行斷線
我使用Fun-ASR-Nano中的即時轉錄serve_realtime_ws.py的腳本，但目前遇到個問題是當我連線開始錄音後，大約連線一段時間後就會斷線，尤其是我有開啟說話者識別的時候，大約在20~30分鐘之間就會斷線了，不曉得各位是否有遇到類似的問題，又應該如何解決？

是透過docker來架設環境:
- OS: ubuntu 
- Python version: 3.12.3
- FunASR version: 1.3.14
- ModelScope version: 1.37.1
- PyTorch / torchaudio version: 2.10.0
- Install method (`pip`, source, Docker): Docker
- Device (`cuda`, `cpu`, `mps`): cuda 
- GPU model: NVIDIA 6000ada 48GB
- CUDA/cuDNN version: 13.2

### PR DESCRIPTION
**Pull request by LauraGPT**
## Summary

- retain only a rolling audio window while keeping absolute session timestamps
- cap speaker chunk/embedding history and preserve stable speaker IDs with bounded centroids
- move blocking session work off the asyncio loop while serializing access to shared models
- release raw audio on finalization, including sub-chunk sessions

Fixes #3101

## Validation

- `python3 -m pytest tests/test_realtime_ws_service.py -q` (`13 passed`)
- real `funasr`/SciPy/sklearn speaker clustering smoke test (`8` chunks, stable speaker `0`)
- synthetic 2-hour, 16 kHz stream: `72,000` frames in `1.640s`; retained buffer `80,000` samples / `320,000` bytes; reported duration `7,200,000ms`
- `git diff --check`
- `python3 -m py_compile examples/industrial_data_pretraining/fun_asr_nano/serve_realtime_ws.py tests/test_realtime_ws_service.py`

### CODE REVIEW

File: examples/industrial_data_pretraining/fun_asr_nano/serve_realtime_ws.py [hunk 1]
```diff
@@ -77,85 +78,140 @@ def _clean_asr_text(text):
...
         final_sentences = []
         for s in sentences:
-            sub = self._try_split(s, sv_output, id_map, min_split_s)
-            final_sentences.extend(sub)
+            if s["end"] <= history_start_ms:
+                final_sentences.append(s)
+                continue
+            current = dict(s)
+            self.distribute_spk([current], sv_output)
+            final_sentences.extend(self._try_split(current, sv_output, {}, min_split_s))
```
[the remark this answers is not in the document]
Comment (LauraGPT) [reply examples/industrial_data_pretraining/fun_asr_nano/serve_realtime_ws.py:234]: Fixed in d51cf1631e89d633c6ae00e7066b56965fd939df. Sentences that cross the retained-history boundary are now split at that boundary: the prefix keeps its original speaker ID, and only the suffix is redistributed/re-split against the recent speaker window. If the text cannot be split without producing an empty side, the original sentence is preserved.

A regression test reproduces the previous whole-sentence reassignment and now verifies both fragments and speaker IDs. The 13 targeted tests pass, and a real FunASR `sv_chunk`/`postprocess` smoke test verified a 21.5s history boundary with no text loss.
